core/includes/install.core.inc

```diff
@@ -1712,7 +1712,13 @@ function install_finished(&$install_state) {
   // Will also trigger indexing of profile-supplied content or feeds.
   backdrop_cron_run();
 
-  $message = st('Thank you for installing @profile!', array('@profile' => backdrop_install_profile_distribution_name()));
+  $profile = backdrop_install_profile_distribution_name();
+  $message = st('Thank you for installing @profile!', array('@profile' => $profile));
+
+  // Confirming that the Dashboard module is enabled. 
+  if (module_exists('dashboard')) {
```

Comment:
Do we also need to check access to the dashboard? Or can we assume that the person who just istalled the site is user 1 and will have access to everything?